lux/core/frame.py

```diff
@@ -690,7 +690,11 @@ def current_vis_to_JSON(vlist, input_current_vis=""):
         current_vis_spec = {}
         numVC = len(vlist)  # number of visualizations in the vis list
         if numVC == 1:
-            current_vis_spec = vlist[0].to_code(language=lux.config.plotting_backend, prettyOutput=False)
+            plot_backend = lux.config.plotting_backend
```

Comment:
To avoid duplicate code, I cleaned up this logic so that we make the change uniformly when we set plotting backend.